### PR TITLE
auto ocp-69968

### PR DESCRIPTION
**Pull request by jiajliu**

### CODE REVIEW

**Comment by jiajliu** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-stable-4.15-upgrade-from-stable-4.15-aws-ipi-disc-priv-tp-f28

**Comment by jiajliu** [comment]
cc @jianlinliu @shellyyang1989 for a review, thanks!

File: ci-operator/step-registry/cucushift/upgrade/mirror-images/by-oc-adm/cucushift-upgrade-mirror-images-by-oc-adm-commands.sh [hunk 1]
```diff
@@ -196,7 +196,7 @@ do
 
     extract_oc
 
-    if check_ota_case_enabled "OCP-30832"; then
+    if check_ota_case_enabled "OCP-30832" "OCP-69968"; then
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/upgrade/mirror-images/by-oc-adm/cucushift-upgrade-mirror-images-by-oc-adm-commands.sh:199]: This if block is for `OCP-30832` case, mirror payload to a custom repo, and create signature together when mirroring, `OCP-69968` also need the steps?

Comment (jiajliu) [reply ci-operator/step-registry/cucushift/upgrade/mirror-images/by-oc-adm/cucushift-upgrade-mirror-images-by-oc-adm-commands.sh:199]: yes, ocp-69968 needs a mirror with signature together.

**Comment by jianlinliu** [comment]
cc @liangxia to review.

**Pull request merged**